### PR TITLE
media-libs/harfbuzz: add experimental useflag

### PR DESCRIPTION
**Pull request by Alessandro-Barbieri**
EAPI 8
bump python implementations

Closes: https://bugs.gentoo.org/800109
Package-Manager: Portage-3.0.20, Repoman-3.0.3
Signed-off-by: Alessandro Barbieri <lssndrbarbieri@gmail.com>

### CODE REVIEW

File: media-libs/harfbuzz/metadata.xml [hunk 1]
```diff
@@ -17,7 +17,11 @@
...
 	</upstream>
 	<use>
-		<flag name="glib">Compile with GLib Unicode support</flag>
-		<flag name="graphite">Use graphite to render complex non-Roman writing systems</flag>
+		<flag name="cairo">Use <pkg>x11-libs/cairo</pkg> graphics library</flag>
+		<flag name="experimental">Enable experimental APIs</flag>
+		<flag name="glib">Enable <pkg>dev-libs/glib</pkg> unicode functions</flag>
+		<flag name="graphite">Enable <pkg>media-gfx/graphite2</pkg> complementary shaper</flag>
+		<flag name="icu">Enable <pkg>dev-libs/icu</pkg> library unicode functions</flag>
+		<flag name="introspection">Enable <pkg>dev-libs/gobject-introspection</pkg> bindings</flag>
```
Comment (Polynomial-C) [inline media-libs/harfbuzz/metadata.xml:25]: There's no need to add `USE` descriptions for flags that already have a global description. Please only add `USE` flag descriptions for flags where `repoman` or `pkgdev` complains about missing `USE` description.

File: media-libs/harfbuzz/harfbuzz-9999.ebuild [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{8..10} pypy3 )
```
Comment (Polynomial-C) [inline media-libs/harfbuzz/harfbuzz-9999.ebuild:6]: Did you test the package with `python3_10` and `pypy3`? If not, please do not add these.

**Review by Polynomial-C — CHANGES_REQUESTED** [review]
See my individual comments

File: media-libs/harfbuzz/harfbuzz-9999.ebuild (hunk 2 above)
Comment (Alessandro-Barbieri) [reply media-libs/harfbuzz/harfbuzz-9999.ebuild]: python10 work, pypy3 isn't used by meson

**Comment by Alessandro-Barbieri** [comment]
@Polynomial-C is all ok now?

**Comment by Polynomial-C** [comment]
Sorry for the delay. I've split your commit into two. The first one contains the changes you did to the live ebuild and the second commit contains the revbump to `harfbuzz-2.8.2-r1`.

**Pull request closed without merging**